policies/gac/gac_networks.py

```diff
@@ -101,6 +101,8 @@ def __call__(self, state, taus, actions=None):
         state_embedding = tf.expand_dims(self.state_embedding(state), 1)
         # batch x action dim x 400
         noise_embedding = self.noise_embedding(taus)
+        print(noise_embedding.shape)
+        print(self.action_dim)
 
```

Comment:
Also, this branch may require you to rebase onto master before merging. There were some modifications I made in order to correct some bugs I discovered while testing.

Reply:
Yeah. I think after I merged master into this branch, the prints are gone.